### DISCUSSION TITLE
permission denied

### DISCUSSION THREAD

**Discussion by ersteinmal** [entropia/tip-toi-reveng#301] [Q&A]
Dear All.
Thank you for your job! 
May Aim was to look into the "Weihnachtsbriefmarke2024.gme" of germany post office. But buy I get 
`C:\Program Files (x86)\Mulitmedia\Tiptoi\tttool-1.11>tttool media Weihnachtsbriefmarke2024.gme
Audio Table entries: 35
tttool: media: CreateDirectory "\\\\?\\C:\\Program Files (x86)\\Mulitmedia\\Tiptoi\\tttool-1.11\\media": permission denied (Zugriff verweigert)`
That might be a message from the tttool - but is it not described.

Does anybody knows the reason? 
Regards Carsten

**Comment by nomeata** [comment]
I'm not an expert on Windows. Maybe you extracted the files as administrator and ran them otherwise?

Maybe someone else can help. 

Also the mailing list at https://lists.nomeata.de/mailman/listinfo/tiptoi is a good place to ask for help.

**Comment by KOTRET** [comment]
Windows requires elevated permissions when writing to folders/files within program / programx86-folder.
to avoid running the cmd in admin mode, either add your user to the `C:\\Program Files (x86)\\Mulitmedia\\Tiptoi\\tttool-1.11`-folder with full permissions or just place the tool somewhere else outside the program-files-folder.
Additionally, windows marks executables and dlls coming from unknown source in order to prevent accidental execution. To avoid this, unzip to your desktop and move the files afterwards instead of directly unzipping to the program files folder.
